### ISSUE TITLE
Full Accessibility & 508 Office Audit COVID-19 Screener

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#9372]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [x] Practice area reviewers create sub-issues attached to this one with feedback for your team on *feature name*
- [x] **Requesting team** completes attached feedback tickets based on definition of done included in each practice area feedback ticket (may differ for each practice area, please refer to attached tickets for further instruction)
- [x] All requesting team attendees complete brief [VSP Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey

## Artifacts - _please provide the following artifacts when you request a full review_
- Access information for the tool or feature in staging
https://staging.va.gov/covid19screen

#9330 - this issue contains myriad design docs for this project

NOTE THIS IS MOBILE ONLY . While it is best practice to work properly with desktop readers and such, this is not the use case for this app. Vets will answer these questions from their phones in the car or as they are walking into the facility. 

- Link to your accessibility test cases:
  - Certify that the following required accessibility tests are complete by checking each of these boxes (your accessibility specialist is available to help). Recommended items are optional, but highly encouraged.
  
    Required:
    - [ ] Read the [accessibility staging review documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) 
    - [x] Conduct axe browser scan, and document or address issues found 
    - [x] Ensure end-to-end tests - including the axe check, pass 
    - [x] Check color pairing for proper contrast 
    - [x] Check for colorblindness issues 
    - [x] Zoom to 200% and confirm readability, no info cut off, etc 
    - [x] Repeat for 300% 
    - [ ] ~Navigate all pages and core functionality with keyboard only (tough for us to QA without additional tools, we can probably use the connecting mobile device to computer route to validate)~
    - [x] Test all pages and core functionality happy path with MacOs + VoiceOver and/or Android + TalkBack

       
    Recommended:
    - [x] Read the [accessibility post-launch processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/post-launch-audit-processes.md) 
    - [ ] ~Add keyboard end-to-end tests in CI/CD test runs~
    - [ ] Conduct axe-coconut browser scan, document or address issues found 
    - [ ] ~Test with [WAVE tool](https://wave.webaim.org/)  (not required for mobile-only)~
    - [ ] ~Zoom to 400% (not required for mobile-only)~

## Additional Notes
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable)

**Comment by artsymartha68** [comment]
#9330 

NOTE THIS IS MOBILE ONLY . While it is best practice to work properly with desktop readers and such, this is not the use case for this app. Vets will answer these questions from their phones in the car or as they are walking into the facility. 

- Link to your accessibility test cases:
  - Certify that the following required accessibility tests are complete by checking each of these boxes (your accessibility specialist is available to help). Recommended items are optional, but highly encouraged.
  
    Required:
    - [ ] Read the [accessibility staging review documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) 
    - [ ] Conduct axe browser scan, and document or address issues found 
    - [ ] Ensure end-to-end tests - including the axe check, pass 
    - [ ] Check color pairing for proper contrast 
    - [ ] Check for colorblindness issues 
    - [ ] Zoom to 200% at 1280px wide and confirm readability, no info cut off, etc 
    - [ ] Repeat for 300% 
    - [ ] Repeat for 400% 
    - [ ] Navigate all pages and core functionality with keyboard only 
    - [ ] Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox 
    
    Recommended:
    - [ ] Read the [accessibility post-launch processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/post-launch-audit-processes.md) 
    - [ ] Add keyboard end-to-end tests in CI/CD test runs 
    - [ ] Conduct axe-coconut browser scan, document or address issues found 
    - [ ] Test with [WAVE tool](https://wave.webaim.org/)

**Comment by artsymartha68** [comment]
Leaving this open for @mchelen or @Yuffster-usds or @stephenBarrs re: axe browser scanning. Other than that, the open individual issues should cover all a11y specifics.

**Comment by mchelen-gov** [comment]
I get 7 total aXe warnings, none of which are due to our app and all occur on va.gov homepage as well. FWIW aXe check also passes in E2E tests https://github.com/department-of-veterans-affairs/vets-website/pull/12994

**Comment by batemapf** [comment]
@jenstrickland @1Copenut can this be closed? I'm not sure what the process is here and how this ticket relates to #9497.

**Comment by jenstrickland** [comment]
We're waiting on the semantic mark-up fixes before I send this to the VA 508 Office, per [discussion in Slack](https://dsva.slack.com/archives/C013WDER4FP/p1594045121260600).

**Comment by mchelen-gov** [comment]
@jenstrickland Let's go ahead and submit this for 508 review and see what they come back with.

**Comment by jenstrickland** [comment]
@mchelen-gov It's in my backlog. 

@1Copenut — Am I still allowed to complete and submit this?

**Comment by 1Copenut** [comment]
@jenstrickland Yes, please proceed as normal.

**Comment by noahgelman** [comment]
The mobile 508 accessibility audit is complete

**Comment by 1Copenut** [comment]
@noahgelman Thank you for the fast and steady work!

/cc @batemapf
